Qapple/Qapple/SourceCode/Feature/3.Comment/3-1.CommentList/CommentFeature.swift

```diff
@@ -15,7 +15,7 @@ import ComposableArchitecture
 struct CommentFeature {
     @ObservableState
     struct State: Equatable {
-        var post: Post = samplePost
+        var post: BulletinBoard = samplePost
```

Comment:
commentFeature post 모델 수정했습니다!
```suggestion
private let samplePost = BulletinBoard(
    id: 1,
```

Reply:
시몬스가 사용하신 Github의 Suggestion 기능은 요렇게 바꿀 건데 내가 미리 작성해뒀으니 커밋만 하면 돼~ 라는 부분이라 요런 경우에서는 그냥 드래그 + 코멘트만 달아주셔도 좋을 것 같아요!

Reply:
오 그렇군요! 알겠습니닷!